modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -361,6 +365,55 @@ curl -v -X POST http://10.143.201.101:8091/pools/default/buckets/testBucket \
 If successful, the call returns a `200 OK` notification.
 No object is returned.
 
+
+[[durabilityImpossibleFallback]]
+=== durabilityImpossibleFallback
+
+Overrides Couchbase Server's default behavior when it cannot meet a durable write's majority requirement.
+When set to the default `disabled` setting, Couchbase Server reports to clients that a durable write that cannot meet its majority requirement has failed.
+It also rolls back any data changes by the write across all nodes in the cluster.
```

Comment:
It does not roll back anything. This check is performed before even attempting to contact a replica based on the configuration that the cluster manager passes to the data service.